web-components/src/global.css.js

```diff
@@ -251,43 +252,50 @@ merch-card [slot='heading-xl'] {
     color: var(--merch-color-grey-80);
 }
 
-merch-card [slot='callout-text'] {
-    display: inline-block;
+merch-card [slot='callout-content'] {
+    display: flex;
+    flex-direction: column;
     margin: var(--consonant-merch-spacing-xxxs) 0px;
+    gap: var(--consonant-merch-card-callout-spacing-xxs);
 }
 
-merch-card [slot='callout-text'] > div {
-    position: relative;
-    display: inline-grid;
+merch-card [slot='callout-content'] > div {
+    display: flex;
+    flex-direction: column;
+    margin: var(--consonant-merch-spacing-xxxs) 0px;
+    gap: var(--consonant-merch-card-callout-spacing-xxs);
+    align-items: flex-start;
+}
+
+merch-card [slot='callout-content'] > div > div {
+    display: flex;
     background: rgba(203 203 203 / 50%);
     border-radius: var(--consonant-merch-spacing-xxxs);
     padding: var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxxs) var(--consonant-merch-spacing-xxs);
 }
 
-merch-card [slot='callout-text'] > div.callout-content-wrapper-with-icon {
-    padding-right: var(--consonant-merch-card-callout-icon-padding);
-}
-
-merch-card [slot='callout-text'] > div > div {
+merch-card [slot='callout-content'] > div > div > div {
```

Comment:
shouldn't we add a class to that last div and avoid rather brittle div > div > div rule?

Reply:
Agreed. Should I also then assign a class to the 2nd last div. I am thinking of something like this:
```
<div slot="callout-content">
 <div>
  <div class="callout-wrapper">
    <div class="callout-text">
    <img>
```